product_docs/docs/biganimal/release/administering_cluster/01_portal_access.mdx

```diff
@@ -5,129 +5,87 @@ redirects:
     - 01_user_access
 ---
 
+BigAnimal uses role-based access controls to grant users access to different parts of the application using organization-level and project-level roles. Roles are sets of permissions. Roles are used to manage permissions assigned to users.
 
-BigAnimal uses role-based access controls to grant users access to different parts of the application.
+Each customer has a unique BigAnimal organization. Each organization has at least one project by default; more projects can be created. An organization's database clusters are deployed and managed in the customer's cloud account within a project. Projects within an organization separate workflows to provide secure and isolated environments to ensure complete separation of customer data between different projects within an organization.
```

Comment:
I changed the following sentence to present tense. I don't think we need future tense here. We try to avoid future tense in the docs. I can change it back if you disagree.
```suggestion
Each organization has at least one project by default; more projects can be created. 
```

Reply:
see https://github.com/EnterpriseDB/docs/pull/3346/commits/e626e7ee18fccbfac5ef40ec00f9c3d12ef0c496 for my changes